R/aplot.R

```diff
@@ -0,0 +1,27 @@
+##' @method print aplot
+##' @importFrom patchwork plot_layout
+##' @importFrom patchwork plot_spacer
+##' @importFrom ggplot2 ggplot
+##' @importFrom ggplot2 theme_void
+##' @export
+print.aplot <- function(x, ...) {
+  grid.draw(x)
+}
+
+
+##' @importFrom grid grid.draw
+##' @importFrom aplot as.patchwork
+##' @method grid.draw aplot
+##' @export
+grid.draw.aplot <- function(x, recoding = TRUE) {
+  attr(x,"class") <- "aplot"
+  grid::grid.draw(as.patchwork(x,align="y"))
+}
+
+##' @importFrom ggplot2 ggplotGrob
+##' @importFrom patchwork patchworkGrob
+##' @importFrom aplot as.patchwork
+aplotGrob <- function(x) {
+  res <- as.patchwork(x,align="y")
+  patchworkGrob(res)
+}
```

Comment:
no need to copy codes from aplot.

Reply:
These codes are not the same as the codes from aplot, it overwrited the codes.
https://github.com/YuLab-SMU/aplot/blob/156665cd9c0ab2775cd3547268d94ff915df33e9/R/aplot.R#L93-L105

These function add additional parameter `align = "y"` to `as.patchwork()`.

if these parameter are not added, the plot above will not align to the plot below and the coordinate below will be changed.
![t5(1)](https://user-images.githubusercontent.com/78794151/194734925-b1a3c0c4-d676-4b60-8ae5-8127795b9058.png)

Reply:
the issue was solved by aplot v=0.1.8 and this commit, <https://github.com/YuLab-SMU/ChIPseeker/commit/6161ead8a0497b9d9302d6be114640094b68a39f>.